reporters/GlobalSummaryReporter.js

```diff
@@ -0,0 +1,481 @@
+          uniqueLiquidationsPeriod[event.sponsor] = true;
+        }
+      }
+      allLiquidationStatsTable = {
+        ["# of liquidations"]: {
+          cumulative: Object.keys(uniqueLiquidations).length,
+          [this.periodLabelInHours]: Object.keys(uniqueLiquidationsPeriod).length
+        },
+        ["tokens liquidated"]: {
+          cumulative: this.formatDecimalString(tokensLiquidated),
+          [this.periodLabelInHours]: this.formatDecimalString(tokensLiquidatedPeriod)
+        },
+        ["collateral liquidated"]: {
+          cumulative: this.formatDecimalString(collateralLiquidated),
+          [this.periodLabelInHours]: this.formatDecimalString(collateralLiquidatedPeriod),
+          current: this.formatDecimalString(this.collateralLockedInLiquidations)
+        }
+      };
+
+      console.table(allLiquidationStatsTable);
+    }
+  };
+
+  _generateDisputeStats = async () => {
+    const { toBN } = this.web3.utils;
+
+    let allDisputeStatsTable = {};
+
+    let uniqueDisputes = {};
+    let uniqueDisputesPeriod = {};
+    let tokensDisputed = toBN("0");
+    let tokensDisputedPeriod = toBN("0");
+    let collateralDisputed = toBN("0");
+    let collateralDisputedPeriod = toBN("0");
+    let disputesResolved = {};
+
+    if (this.disputeEvents.length === 0) {
+      console.log(dim("\tNo dispute events found for this EMP."));
+    } else {
+      for (let event of this.disputeEvents) {
+        // Fetch disputed collateral & token amounts from corresponding liquidation event that with same ID and sponsor.
+        const liquidationData = this.liquidationEvents.filter(
+          e => e.liquidationId === event.liquidationId && e.sponsor === event.sponsor
+        );
+        tokensDisputed = tokensDisputed.add(toBN(liquidationData.tokensOutstanding));
+        collateralDisputed = collateralDisputed.add(toBN(liquidationData.lockedCollateral));
+        uniqueDisputes[event.sponsor] = true;
+        if (event.blockNumber >= this.startBlockNumberForPeriod) {
+          tokensDisputedDaily = tokensDisputedPeriod.add(toBN(liquidationData.tokensOutstanding));
+          collateralDisputedDaily = collateralDisputedPeriod.add(toBN(liquidationData.lockedCollateral));
+          uniqueDisputesPeriod[event.sponsor] = true;
+        }
+
+        // Create list of resolved prices for disputed liquidations.
+        try {
+          const liquidationTimestamp = (await this.web3.eth.getBlock(event.blockNumber)).timestamp;
+          const hasResolvedPrice = await this.oracleContract.hasPrice(
+            await this.empContract.methods.priceIdentifier().call(),
+            liquidationTimestamp,
+            {
+              from: this.empContract.options.address
+            }
+          );
+          if (hasResolvedPrice) {
+            // @dev: We need to check `hasPrice` first because `getPrice` as called below will return a non-sensical, really high
```

Comment:
@mrice32 highlighting this "bug" I found using web3 to query `oracle.getPrice()`

Reply:
Resolved this as discussed IRL by adding a `revertWrapper` function to the common modules that will check for this false positive